app/src/main/res/values/strings.xml

```diff
@@ -34,5 +34,7 @@
     <string name="title_event_detail">Event Detail</string>
     <string name="subtitle_organizer">Organizer: %s</string>
     <string name="event_thumbnail">Event Thumbnail</string>
+    <string name="text_empty">Nothing to show here</string>
```

Comment:
"This is general message to be shown in both event and attendee layout. Please suggest what else it can be"